websocket/2.2/_index.md

```diff
@@ -9,35 +9,34 @@ This release removes references to the SecurityManager and provides some minor u
+
+* Add the `getSession()` method to `SendResult`.
+
+* Clarify the behaviour if `@OnMessage.maxMessageSize` it is set to a value larger than `Integer.MAX_VALUE`.
 
 ### Removals, deprecations or backwards incompatible changes
-* All references to the SecurityManager have been removed
+* None
 
 ### Minimum Java SE Version
 **Java SE 8 or higher**
 
 # Details
-* [Jakarta WebSocket 2.2 Release Record](https://projects.eclipse.org/projects/ee4j.websocket/releases/2.2.0)
-
-<!--
-The following can be uncommented and version information updated as they become available.
-
-    * [Jakarta EE Platform 10 Release Plan](https://jakartaee.github.io/platform/jakartaee10/JakartaEE10ReleasePlan)
-* [Jakarta WebSocket 2.1 Specification Document](./jakarta-websocket-spec-2.1.pdf) (PDF)
-* [Jakarta WebSocket 2.1 Specification Document](./jakarta-websocket-spec-2.1.html) (HTML)
+* [Jakarta WebSocket 2.2 Release Plan](https://projects.eclipse.org/projects/ee4j.websocket/releases/2.2.0/plan)
+* [Jakarta EE Platform 11 Release Plan](https://jakartaee.github.io/platform/jakartaee11/JakartaEE11ReleasePlan)
+* [Jakarta WebSocket 2.2 Specification Document](./jakarta-websocket-spec-2.2.pdf) (PDF)
+* [Jakarta WebSocket 2.2 Specification Document](./jakarta-websocket-spec-2.2.html) (HTML)
 * [Jakarta WebSocket 2.1 Javadoc]
     * [Client](./apidocs/client)
     * [Server](./apidocs/server)
-* [Jakarta WebSocket 2.1 TCK](https://download.eclipse.org/jakartaee/websocket/2.1/jakarta-websocket-tck-2.1.0.zip)  ([sig](https://download.eclipse.org/jakartaee/websocket/2.1/jakarta-websocket-tck-2.1.0.zip.sig),  [sha](https://download.eclipse.org/jakartaee/websocket/2.1/jakarta-websocket-tck-2.1.0.zip.sha256),  [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* [Jakarta WebSocket 2.2 TCK](https://www.eclipse.org/downloads/download.php?file=/ee4j/jakartaee-tck/jakartaee11/staged/eftl/jakarta-websocket-tck-2.2.0.zip) ([sig](tbd), [sha](https://www.eclipse.org/downloads/download.php?file=/ee4j/jakartaee-tck/jakartaee11/staged/eftl/jakarta-websocket-tck-2.2.0.zip#btn-ajax-checksum-sha512), [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-    * [jakarta.websocket:jakarta.websocket-api:jar:2.1.0](https://search.maven.org/artifact/jakarta.websocket/jakarta.websocket-api/2.1.0/jar)
-    * [jakarta.websocket:jakarta.websocket-client-api:jar:2.1.0](https://search.maven.org/artifact/jakarta.websocket/jakarta.websocket-client-api/2.1.0/jar)
--->
+    * [jakarta.websocket:jakarta.websocket-api:jar:2.2.0](https://search.maven.org/artifact/jakarta.websocket/jakarta.websocket-api/2.2.0/jar)
+    * [jakarta.websocket:jakarta.websocket-client-api:jar:2.2.0](https://search.maven.org/artifact/jakarta.websocket/jakarta.websocket-client-api/2.2.0/jar)
 
 # Compatible Implementations
 
-* Eclipse Tyrus (TBC)
+* Apache Tomcat 11.0.0-M19-SNAPSHOT
```

Comment:
This should be a download link, not just a name.
The snapshot link on the CCR no longer seems to be valid

Reply:
@markt-asf   
Thanks for updating PR.

* The URL of Tyrus is TCK result, but not Tyrus download link. Please change it to the download link.
* Please also update CCR links of this PR template to the new one https://github.com/jakartaee/websocket/issues/427,
and add Tyrus CCR.

Reply:
@markt-asf   
Thanks for updating Tyrus links.

The CCR links you added are not valid for this PR, because the TCK SHA values are different form TCK binary in this PR.  
So, Tyrus CCR are needed to update as well as Tomcat M20.

Reply:
The PR uses the SHA512 since that is the only hash that the download page presents. The SHA-512 is:
5bb92094db8d46f60f561e0bcb81146b4802072abc39ce059bb69a63b310d3e2c970ebf8b1a5fbcc97f8410f4d05918ac2b21c847d89af027e9191d0e7e45a20

This is the same as the SHA-512 listed on the Tomcat CCR.

The SHA-256 is (obtained by downloading the file, confirming the SHA-512 hash and the calculating the SHA-256 hash):
ff50825305bd1318a4e6bdfab21b829cfd8cf89fab8a970f007b52636d3f2629

The Tyrus CCR references the correct SHA-256 hash.

I am not seeing any inconsistencies here apart from the download page presenting the SHA-512 hash but the PR checklist expecting the SHA-256 hash. The WebSocket project has approved the Tyrus CCR having confirmed the SHA-256 hash.

I do not see any changes that are required for this to proceed.

Reply:
@markt-asf   
I didn't notice the difference SHA-256 and SHA-512. Thanks for clarification.